ArduPlane/radio.cpp

```diff
@@ -145,7 +145,7 @@ void Plane::read_radio()
         && channel_throttle->get_control_in() > 50
         && stickmixing) {
         float nudge = (channel_throttle->get_control_in() - 50) * 0.02f;
-        if (ahrs.using_airspeed_sensor()) {
+        if (TECS_controller.use_airspeed()) {
```

Comment:
So essentially we wouldn't be able to airspeed nudge if we were using synthetic airspeed. Sure, this one looks like an improvement.